plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -3439,6 +3438,22 @@ public void testIfDeletesReturnsNumberOfRemovedRows()
         assertUpdate("DELETE FROM " + tableName + " WHERE key = 'two'", 2);
     }
 
+    @Test
+    public void testUpdatingFileFormat()
+    {
+        String tableName = "test_updating_file_format_" + randomTableSuffix();
+
+        assertUpdate("CREATE TABLE " + tableName + " WITH (format = 'orc') AS SELECT * FROM nation WHERE nationkey < 10", "SELECT count(*) FROM nation WHERE nationkey < 10");
```

Comment:
Please use also  https://trino.io/docs/current/connector/iceberg.html#properties-table for reading the properties of the Iceberg table.